src/mds/Server.cc

```diff
@@ -349,10 +349,17 @@ void Server::handle_client_session(MClientSession *m)
 	m->put();
 	return;
       }
+      // This looks as if we are getting a seq that is higher than
+      // expected.  I didn't notice any "wrap" handling, but the seqn is
+      // 64-bits ...
```

Comment:
adding more log info, editing code comments, will update commit and "squash" them if I fiigure out how